nifi-nar-bundles/nifi-ocr-bundle/nifi-ocr-processors/src/test/resources/tessdata/tessconfigs/nobatch

```diff
@@ -0,0 +1 @@
+
```

Comment:
@mattyb149 this is an empty configuration file. It can be used in conjunction with whitelisting of detected words so I thought it would make sense to leave it there in case someone wanted to add unit testing around those capabilities at a later time.